centreon/www/front_src/src/Login/useLogin.ts

```diff
@@ -206,15 +201,9 @@ const useLogin = (): UseLoginState => {
     );
   }, []);
 
-  const forcedProviders = useMemo(
-    () => getForcedProviders(providers || []),
-    useDeepCompare([providers])
-  );
+  const forcedProviders = getForcedProviders(providers || []);
```

Comment:
what issue the memoization caused ?

Reply:
@Noha-ElAbrouki 
I don't know, this is the revert because it was a bad cherry-pick